pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -438,6 +438,9 @@ func (r *ReconcileLogStorage) Reconcile(ctx context.Context, request reconcile.R
 		hdler = utils.NewComponentHandler(reqLogger, r.client, r.scheme, managementClusterConnection)
 	}
 
+	//Handler for the ECK ES public secret
+	hdlrPublicSecret := utils.NewComponentHandler(reqLogger, r.client, r.scheme, nil)
```

Comment:
`hdlrPublicSecret` is what we're using it for but I think a better name would be `hdlrNoOwner`.

Reply:
Renamed to `hndlerNoOwner`.